common/test/acceptance/pages/studio/users.py

```diff
@@ -78,6 +80,48 @@ def click_submit_new_user_form(self):
         self.q(css='.form-create.create-user .action-primary').click()
         wait_for_ajax_or_reload(self.browser)
 
+    def get_user(self, email):
+        """ Gets user wrapper by email """
+        target_users = [user for user in self.users if user.email == email]
+        assert len(target_users) == 1
```

Comment:
@cahrens the intention here was to emphasize that (1) it is not a test assertion and (2) there should be single user. I believe if for some reason there would be two users, assertion error thrown here would make more sense than possibly unwanted behavior if further actions are performed on wrong user.